sql-cli/tests/test_project.py

```diff
@@ -1,29 +1,22 @@
-import tempfile
+from tempfile import TemporaryDirectory
```

Comment:
This feels like a matter of style

Reply:
Yes. Are you okay with that change?

Reply:
It feels unnecessary, but I'm okay :)

Reply:
I found an even better way then using the `tempfile.TemporaryDirectory` see 5ee08b7 :)